sklearn/multioutput.py

```diff
@@ -0,0 +1,221 @@
+            Multi-output targets. An indicator matrix turns on multilabel
+            classification.
+            Predicted multi-class targets across multiple predictors.
+            Note:  entirely separate models are generated
+            for each predictor.
+        """
+        check_is_fitted(self, 'estimators_')
+
+        results = np.column_stack([estimator.predict(X) for estimator in
+                                   self.estimators_])
+        return results
+
+    def predict_proba(self, X):
+        """Probability estimates. This returns prediction probabilites
+        for each class for each label.
+
+        Parameters
+        ----------
+        X : array-like, shape = [n_samples, n_features]
+
+        Returns
+        -------
+        T : (sparse) array-like, shape = [n_samples, n_classes, n_outputs]
+            The class probabilities of the samples for each of the outputs
+        """
+        check_is_fitted(self, 'estimators_')
+
+        results = np.dstack([estimator.predict_proba(X) for estimator in
+                            self.estimators_])
+        return results
+
+    def score(self, X, y):
+        """"Returns the mean accuracy on the given test data and labels.
+
+        Parameters
+        ----------
+        X : array-like, shape = [n_samples, n_features]
+        y : array-like, shape = [n_samples, n_outputs]
+
+        Returns
+        -------
+        scores : array-like, shape = [n_outputs, 1]
+            Array of p_target floats of the mean accuracy
+            of each estimator_.predict wrt. y.
+        """
+        check_is_fitted(self, 'estimators_')
+        if y.ndim == 1:
+            y = y.reshape(-1, 1)
+        if y.shape[1] != self.n_outputs:
+            raise ValueError("The number of outputs of Y for fit {0} and"
+                             " score {1} should be same".
+                             format(self.n_outputs, y.shape[1]))
+        results = np.row_stack([estimator.score(X, y[:, i]) for i, estimator in
+                                enumerate(self.estimators_)])
+        return results
+
+    @property
+    def multilabel_(self):
+        """returns a vector of whether each classifer is a  multilabel
+        classifier in tuple for
+        """
+        return [(label, model_.multilabel_)
+                for label, model_ in enumerate(self.estimators_)]
+
+    @property
+    def classes_(self):
+        return [(label, model_.label_binarizer_)
+                for label, model_ in enumerate(self.estimators_)]
+
+    @property
+    def coef_(self):
+        check_is_fitted(self, 'estimators_')
+        if not hasattr(self.estimators_[0], "coef_"):
+            raise AttributeError(
+                "Base estimator doesn't have a coef_ attribute.")
+        coefs = [e.coef_ for e in self.estimators_]
+        if sp.issparse(coefs[0]):
+            return sp.vstack(coefs)
+        return np.vstack(coefs)
+
+    @property
+    def intercept_(self):
```

Comment:
Why do we need these properties. If needed, the user can directly get it from the list of estimators